src/services/specification.service.ts

```diff
@@ -104,6 +107,29 @@ export class SpecificationService {
     return false;
   }
 
+  static shouldInformAboutLatestVersion(
+    version: string,
+  ): boolean {
+    const oneDay = 24 * 60 * 60 * 1000; /* ms */
```

Comment:
Not sure what I feel about this 😄 In theory if you said no once, I think it should not ask again, but discretely show it somewhere, that the document is not up to date 🤔 Thoughts?

Reply:
I don't want to show it on every time, so it shows once a day (or once a session) - just like in playground. I can show information that there is a new version in the terminal, next to `Valid/Invalid` info, that the document does not use the latest version. What do you think?

Reply:
The terminal part sounds like a good idea 👍 

I don't think it should re-ask each day though, once is enough, but that's just my preference 😄

Reply:
![image](https://user-images.githubusercontent.com/20404945/136060039-6805b0ef-96f9-439b-adbd-408ef5303a27.png)

Reply:
Can you add a red negative sign, opposite of `valid` basically, and make it so you can click it to convert? 🤔

Reply:
Just want the `NOT LATEST` to be more visible to the user 🙂

Reply:
Ok, I will do it tomorrow :) Thanks for suggestions! `NOT LATEST` will have warning sign.